### PR TITLE
Updated revhistory entries to unify the structure, check dates, titles, punctuation

### PR DESCRIPTION
**Pull request by dariavladykina**
### Description

Revised revision history entries in Smart Docs per the new template (https://github.com/openSUSE/doc-kit/pull/101).

### CODE REVIEW

**Comment by dariavladykina** [comment]
A comment: The assembly `articles/SELinux-policy.asm.xml` is a stub, I believe it can be removed.

File: articles/elemental.asm.xml [hunk 1]
```diff
@@ -48,14 +48,14 @@
         <revision><date>2023-11-28</date>
           <revdescription>
             <para>
-              Renamed to 'Rancher Guide (Elemental)'.
+              Updated the title to <quote>Rancher Guide (Elemental)</quote>
```
Comment (taroth21) [inline articles/elemental.asm.xml:51]:
```suggestion
              Updated the title to <citetitle>Rancher Guide (Elemental)</citetitle>
```

File: articles/sudo-configure-superuser-privileges.asm.xml [hunk 2]
```diff
@@ -64,18 +64,8 @@
...
-                </itemizedlist>
-              </listitem>
-              <listitem><para>Removed sections:</para>
-                <itemizedlist>
-                  <listitem><para>Removed section on <quote>Using sudo without root password</quote>.</para></listitem>
-                </itemizedlist>
-              </listitem>
-            </itemizedlist>
+            <para>Updated section on <quote>Creating custom sudo configurations</quote>;
+              removed section on <quote>Using sudo without root password</quote></para>
```
Comment (taroth21) [inline articles/sudo-configure-superuser-privileges.asm.xml:68]:
```suggestion
            <para>Updated section on creating custom sudo configurations;
              removed section on using sudo without root password</para>
```

Comment (taroth21) [reply articles/sudo-configure-superuser-privileges.asm.xml:68]: I would rather not quote the exact section titles here but use a descriptive phrase (reason: if the section title is changed later on probably nobody will think of updating the revhistory entry accordingly)

**Review by taroth21 — CHANGES_REQUESTED** [review]
Hi @dariavladykina: Many thanks - just two small nitpicks here (see suggestions), otherwise looks good to me. Many thanks!

**Review by taroth21 — APPROVED** [review]
LGTM, many thanks!

**Pull request merged**